WordPress/src/main/java/org/wordpress/android/ui/main/MySiteFragment.java

```diff
@@ -459,4 +489,15 @@ public void onSiteChanged(OnSiteChanged event) {
         }
         refreshSelectedSiteDetails();
     }
+
+    @SuppressWarnings("unused")
+    @Subscribe(threadMode = ThreadMode.MAIN)
+    public void onPostUploaded(PostStore.OnPostUploaded event) {
+        final PostModel post = event.post;
+        SiteModel site = getSelectedSite();
+        if (isAdded() && event.post != null && event.post.getLocalSiteId() == site.getId()) {
```

Comment:
oops good catch - did that check and also moved `isAdded()` check a bit up here 28a9d58